src/components/Modals/CourseMenu.vue

```diff
@@ -29,6 +29,30 @@
           </div>
         </div>
       </div>
+      <div
+        class="courseMenu-section"
+        @mouseover="setDisplayEditCourseCredits(true)"
+        @mouseleave="setDisplayEditCourseCredits(false)"
+        v-if="this.getCreditRange[0] != this.getCreditRange[1]"
+        >
+        <div class="courseMenu-left">
+          <img class="courseMenu-icon" src="../../assets/images/edit-credits.svg" />
```

Comment:
Better to use the "@" symbole to get to the src directory than "../../"

Reply:
I was using the same filepaths as the other icons should I change those too?